### PR TITLE
Implementing setex, expire (lazy expire check)

### PR DESCRIPTION
**Pull request by clarity0**
Started working on this before i saw your other PR, gonna create this one anyways.

### CODE REVIEW

**Comment by evoxmusic** [comment]
Hey @clarity0, thank you - it's good to compare 2 ways of responding to the same request. @pjeziorowski what do you think of this PR?

File: redisless/src/server.rs [hunk 1]
```diff
@@ -174,6 +174,14 @@ fn run_command_and_get_response<T: Storage>(
                 unlock(storage).set(k.as_slice(), v.as_slice());
                 protocol::OK.to_vec()
             }
+            Command::Setex(k, v, duration) => {
+                unlock(storage).setex(k.as_slice(), v.as_slice(), *duration);
+                protocol::OK.to_vec()
+            }
+            Command::Expire(k, duration) => {
+                unlock(storage).expire(k.as_slice(), *duration);
```
Comment (pjeziorowski) [inline redisless/src/server.rs:182]: Hey :) We should follow Redis API (for the return type for EXPIRE command here) - https://redis.io/commands/expire#return-value

**Comment by pjeziorowski** [comment]
@clarity0 could you also please test your commands with a Redis client here? https://github.com/Qovery/RedisLess/blob/main/redisless/src/server.rs

**Comment by pjeziorowski** [comment]
@evoxmusic LGTM besides the two points I commented

**Comment by clarity0** [comment]
Some things be fixed now, but I cant figure out why I get timeouts from the server when I run the tests, same thing happens when running the tests on the main branch.

**Review by evoxmusic — APPROVED** [review]
LGTM 👍🏽 - what about you @pjeziorowski ?

**Comment by evoxmusic** [comment]
@clarity0 I am going to fix the Redis test implementation problem.

**Pull request merged**